os/posix/save_file.c

```diff
@@ -29,35 +33,42 @@ int open_save_file(const char *path, size_t size, struct save_file *file, int *c
   int fd;
   int my_created;
 
-  // Open the file O_EXCL to see if it exists
-  if ((fd = open(path, O_RDWR | O_CREAT | O_EXCL, 0666)) >= 0)
-    my_created = 1;
+  if (path == NULL) {
+    ptr = calloc(size, 1);
```

Comment:
Clever use of `file->ptr` for either the mapping or a calloc'd "junk" save file! :+1:

Reply:
Ditto to `ptr == NULL` test.